.gitignore

```diff
@@ -1,3 +1,6 @@
+# Redwood
+.redwood/*
```

Comment:
it looks like you intended to ignore the .redwood directory but maybe that needs to be "forgotten" because it seems to have been checked in.